test/app/controllers/users_validation.test.js

```diff
@@ -0,0 +1,190 @@
+const request = require('supertest');
```

Comment:
Are you check the controller user_validation or the schema? Change the name of the directory